src/cli/Create.cpp

```diff
@@ -97,21 +109,24 @@ int Create::execute(const QStringList& arguments)
 
     auto key = QSharedPointer<CompositeKey>::create();
 
-    auto password = Utils::getPasswordFromStdin();
-    if (!password.isNull()) {
-        key->addKey(password);
+    if (parser->isSet(Create::SetPasswordOption)) {
+        auto password = Utils::getPasswordFromStdin();
+        if (!password.isNull()) {
+            key->addKey(password);
+        }
     }
 
-    QSharedPointer<FileKey> fileKey;
-    if (parser->isSet(Command::KeyFileOption)) {
-        if (!loadFileKey(parser->value(Command::KeyFileOption), fileKey)) {
+    if (parser->isSet(Create::SetKeyFileOption)) {
+        QSharedPointer<FileKey> fileKey;
```

Comment:
Egad, the name swap is killing me here. Can we standardize on `keyFile` please.

Reply:
all the keys are named like that (`PasswordKey`, `CompositeKey`, etc). You would rename the CLI option?

Reply:
You can use fileKey